Cargo.toml

```diff
@@ -37,7 +37,8 @@ cargo-args = ["-Zunstable-options", "-Zrustdoc-scrape-examples"]
 [dependencies]
 bitflags = "2.3"
 cassowary = "0.3"
-crossterm = { version = "0.26", optional = true }
+# crossterm = { version = "0.26", optional = true }
+crossterm = { git = "https://github.com/benjajaja/crossterm", rev = "ad84a3829d55eac27cfdb7791b2e34c1732040ad", optional = true }
```

Comment:
I think I'd prefer to wait until this is merged to merge this into ratatui. This is fine as a draft for now however.

You can also set this in a patch section and leave the main dependency in place. See https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html#the-patch-section

```toml
[patch.crates-io]
crossterm = { git = "https://github.com/benjajaja/crossterm", rev = "ad84a3829d55eac27cfdb7791b2e34c1732040ad", optional = true }
```

Reply:
Oooh TIL, that's so cool, thank you!